src/shrubbery/core.clj

```diff
@@ -40,22 +40,22 @@
     (matches? [_ _] true)))
 
 (defn call-count
-  "Given a spy, a keyword method name, and an optional vector of args, return the number of times the spy received
+  "Given a spy, a var, and an optional vector of args, return the number of times the spy received
   the method. If given args, filters the list of calls by matching the given args. Matched args may implement the `Matcher`
   protocol; the default implementation for `Object` is `=`."
-  ([spy method]
-   (-> (calls spy) (get method) (count)))
-  ([spy method args]
-  (->>
-     (get (calls spy) method)
-     (filter #(matches? % args))
+  ([spy var]
+   (-> (calls spy) (get var) (count)))
+  ([spy var args]
+   (->>
+     (get (calls spy) var)
+     (filter #(matches? args %))
      (count))))
 
 (defmacro received?
```

Comment:
Does `received?` still need to be a macro? Looks like it could maybe be shifted to a `defn`, which is great.